mkat_fpga_tests/test_cbf.py

```diff
@@ -234,18 +234,36 @@ def calc_zero_and_nonzero_baselines(nonzero_inputs):
                         zeros.add((inp_i, inp_j))
             return zeros, nonzeros
 
-        #zero_baseline, nonzero_baseline = calc_zero_and_nonzero_baselines(nonzero_inputs)
+        zero_baseline, nonzero_baseline = calc_zero_and_nonzero_baselines(nonzero_inputs)
```

Comment:
Perhaps zero_baseline -> zero_baselines, and nonzero_baselines, indicate that these variables contain multiple baselines.